x11-libs/libxt/libxt-1.1.5.recipe

```diff
@@ -0,0 +1,92 @@
+SUMMARY="X Toolkit Intrinsics library"
+DESCRIPTION="libXt provides the X Toolkit Intrinsics, an abstract widget \
+library upon which other toolkits are based. Xt is the basis for many \
+toolkits, including the Athena widgets (Xaw), and LessTif (a Motif \
+implementation)."
+HOMEPAGE="http://xorg.freedesktop.org/releases/individual/lib/"
```

Comment:
```diff
-HOMEPAGE="http://xorg.freedesktop.org/releases/individual/lib/"
+HOMEPAGE="https://xorg.freedesktop.org/releases/individual/lib/"
```

Reply:
I think you want this instead, @fbrosson ;)

```diff
-HOMEPAGE="http://xorg.freedesktop.org/releases/individual/lib/"
+HOMEPAGE="https://www.x.org/releases/individual/lib/"